support-frontend/assets/helpers/billingPeriods.js

```diff
@@ -28,6 +28,19 @@ function billingPeriodNoun(billingPeriod: BillingPeriod, fixedTerm: boolean = fa
   }
 }
 
+function billingPeriodAdverb(billingPeriod: BillingPeriod) {
+  switch (billingPeriod) {
+    case Annual:
+      return 'Annually';
+    case Quarterly:
+      return 'Quarterly';
+    case SixWeekly:
+      return 'Six weekly';
```

Comment:
I'm not sure if this description works because we don't sell anything with a recurring six weekly billing period only 6 for 6 for which six weeks might work better depending on context. Having said that it might not matter because I don't think this is used for GW is it?

Reply:
This isn't being used for anything besides the test; I added 'Six weekly' purely to cover very option in the `BillingPeriod` type, same as 'Quarterly', which also doesn't apply to digi subs.